calico/felix/config.py

```diff
@@ -163,6 +166,14 @@ def validate_cfg(self):
         self.validate_addr("PluginAddress", self.PLUGIN_ADDR)
         self.validate_addr("ACLAddress", self.ACL_ADDR)
 
+        #*********************************************************************#
+        #* Bind address must be * or an IP address.                          *#
+        #*********************************************************************#
+        if (self.BIND_ADDR != "*" and
+                not common.validate_ipv4_addr(self.BIND_ADDR)):
```

Comment:
Fair enough, but in that case we have another bug, which is that we don't support an IPv6 only backend, but we should.